darwin/future/data_objects/validators.py

```diff
@@ -1,3 +1,16 @@
 def parse_name(name: str) -> str:
+    """
+    A function to parse and validate a name
+
+    Parameters
+    ----------
+    name : str
+        The name to be parsed and validated
+
+    Returns
+    -------
+    str
+        The parsed and validated name
+    """
     assert isinstance(name, str)
```

Comment:
I know this is actually my code but was written when first learning pydantic. Don't actually need these type assertions as pydantic does it already